package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@kentico/kontent-cli",
-  "version": "0.0.7",
+  "version": "0.0.8",
   "description": "Command line interface tool that can be used for generating and running Kontent migration scripts",
   "main": "./lib/index.js",
   "types": "./lib/types/index.d.ts",
   "scripts": {
     "build": "tsc",
     "test": "jest",
-    "lint": "tslint --project tsconfig.json --fix \"src/**/*.ts\" && prettier --write \"src/**/*.ts\""
+    "lint": "tslint --project tsconfig.json --fix \"src/**/*.ts\" && prettier --write \"src/**/*.ts\"",
+    "debug": "node --inspect ./lib/index.js migration run --all -e DEV"
```

Comment:
I am not sure if we want to keep this command or not

Reply:
I don't see any issue with keeping it.